source/codegen/metadata/nifake_non_ivi/functions.py

```diff
@@ -455,7 +455,11 @@
                 'direction': 'in',
                 'enum': 'BeautifulColor',
                 'name': 'colors',
-                'type': 'int32[]'
+                'type': 'int32[]',
```

Comment:
note: Again this is in a fake service, and it's not really necessary, but I didn't want to add a suppression for something that should never happen for real (an array without a size type).